### DISCUSSION TITLE
Open Weekly Meetings

### DISCUSSION THREAD

**Discussion by Laurin-W** [assemblee-virtuelle/activitypods#167] [General]
Hi everyone!

We are having open bi-weekly meetings. In the meetings we demonstrate our current work progress for 20 to 30 minutes and leave room for discussion afterwards.

You are welcome to join if you are interested or want to talk about ActivityPub, Solid or beyond.
We are looking forward to hear of your ideas, questions, and thoughts!

:timer_clock: When? Every other Thursday at 15:00 CET (in even weeks, as of 2025-04-01 )
:pushpin: Where? https://bbb.paquerette.eu/rooms/ass-hvm-tbf-usp/join
:bookmark_tabs: Meeting Notes: https://github.com/assemblee-virtuelle/activitypods/wiki/Meeting-Notes

**Comment by outlaw-dame** [comment]
I need to set up a recurring calendar meeting so I do not miss. Will there be notes taken on what the important topics were of each meeting?

**Reply by srosset81** [reply]
We _should_ take notes... but we did not on this week's meeting, sorry !

**Reply by outlaw-dame** [reply]
No worries, I just was disappointed I was busy and missed this meeting

**Comment by outlaw-dame** [comment]
Was there a meeting today?

**Reply by Laurin-W** [reply]
Hey Damon, yes it took place at 15:00 CET, as usual.
@srosset81 talked a little bit about MastoPods which he started developing this week and showed what's there so far. There'll be a post on that soon as well.
Best, Laurin

**Reply by outlaw-dame** [reply]
Thanks for your response. I got the time mixed up and attended at 10am EST I will put it in my calendar. If you have some time soon to meet briefly I would appreciate that. If not I will do my best to attend next weeks meeting.

**Reply by Laurin-W** [reply]
This week is going to be quite busy for me, I'm not sure about @srosset81.

If you have specific questions or thoughts that you'd like to discuss, maybe you could write them here and we try to respond? :)

**Comment by Laurin-W** [comment]
You can find the meeting notes here:

https://github.com/assemblee-virtuelle/activitypods/wiki/Meeting-Notes
